no-computer.md

```diff
@@ -58,12 +123,37 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Since Diplomacy already has a syntax for submitting instructions, it would be
+best to use the same syntax.
+However, we would need to create an intuitive interface to store the map.
+A map can be easily translated into a graph and 
+there are a lot of ways of representing graphs.
+So I think it would not be too difficult to choose a representation and build
+an interface for entering the map's data.
+
+There would not be a lot of work in sorting out semantics since the game
+has clear rules on which moves are valid and on resolving conflicts.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The project is not very ambitious.
+The bulk of the work would be in designing an interface for entering the map
+into the program.
+Since the DSL has a very clear use, the scope is clear.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+The project would be a good idea because I have a set group of people that
+I would be designing this for.
+On the other hand, this group may be too small and specific. 
+I would not experience making tough decisions on ambiguous naming since I have a
+group I could poll for every design decision.
+
+The project seems to be two DSLs. 
+The first one the game already specified and the second one would be for 
+representing a map.
+So the project may be too simple.
+
```

Comment:
I think I agree that the project may be too simple but there is also the interesting aspect of bringing in the community for help and creating a Bazaar language design space that we read about.